docs/api.md

```diff
@@ -140,6 +140,9 @@ Since this is a HTTP-based protocol, clients should be prepared to gracefully ha
 * Miscellaneous
     * [POST /v1/get_random_bytes](#post-v1get_random_bytes)
 
+* OAuth
+    * [POST /v1/oauth/sessions/revoke (:lock: accessToken)](#post-v1oauthsessionsrevoke)
+    * [POST /v1/oauth/keys/revoke (:lock: accessToken)](#post-v1oauthkeysrevoke)
```

Comment:
I don't think we need to use the word "oauth" in the API path here.  The endpoints are authenticated via oauth, but they're not manipulating any oauth-related resources, so ISTM we'd be exposing an implementation detail in the name.  

Also we might like to keep the `/oauth` prefix free for an eventual grafting of the oauth server API back onto its partner server...

Two additional concerns to factor into the API structure here:
*  let's add a "list all session tokens on the account" endpoint while we're at it
*  will we ever want to revoke a keyFetchToken without also revoking the associated sessionToken?  If not, let's make revoking a sessionToken also revoke the corresponding keyFetchToken in order to reduce the number of ways a caller of the API could do things wrong.  It's more work for us but seems nicer for callers.

Combining the above with your fine work over in https://github.com/mozilla/fxa-oauth-server/pull/262, and trying to stay consistent with the existing API, how about something like:

  * GET /v1/account/sessions
  * GET /v1/session/:id/status
  * POST /v1/session/:id/destroy

@kitcambridge @dannycoates thoughts?  (As @dannycoates can attest, I tend to be very opinionated about API design, so don't hesitate to push back on all this!)